docs/sql/index.md

```diff
@@ -56,8 +56,375 @@ When you return data in CSV or raw format, each row corresponds to a *document*,
 
 ## User interfaces
 
-You can test queries using **Dev Tools** in Kibana (`https://<host>:5601`).
+Kibana
+{: .label .label-yellow :}
 
+### Index data
+
+The SQL plugin is for read-only purposes, so you cannot index or update data using SQL.
+
+Use the `bulk` operation to index some sample data:
+
+```json
+PUT accounts/_bulk?refresh
+{"index":{"_id":"1"}}
```

Comment:
Where's this data from? Just curious. If there's a reference here, I'm not catching it.

Reply:
The sample data is from here - https://github.com/opendistro-for-elasticsearch/sql/blob/master/src/test/resources/doctest/testdata/accounts.json